test-tools/build.gradle.kts

```diff
@@ -136,6 +136,10 @@ tasks.sourcesJar {
 
 publishing {
     publications.withType<MavenPublication> {
+        suppressPomMetadataWarningsFor("apiElements")
```

Comment:
Why suppress suddenly?

Reply:
No reason for the noise; I'm surprised it's not suppressed in the other places we use this construct -- we're aware of the gap and are satisfied with the result we get ... the extra warnings are useless.